src/annotator/sidebar.js

```diff
@@ -53,7 +59,11 @@ function createSidebarIframe(config) {
  * as well as the adjacent controls.
  */
 export default class Sidebar extends Guest {
-  constructor(element, config) {
+  /**
+   * @param {HTMLElement} element
+   * @param {Record<string, any>} config
```

Comment:
I forget the meaningful difference between `Record<string, any>` and `Object<string, any>`. Do you recall off-hand?

Reply:
It is not very significant change.
`Record` is a little bit more stringent than `Object`. For example, in the current setup this reassignment `config = null` doesn't raise an error if `Object` is used, but it raises a type error if `Record` is used.